weave-js/src/common/util/links.tsx

```diff
@@ -115,6 +112,23 @@ export const TargetBlank: FCWithRef<
   })
 );
 
+// Taken from - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href
+const BLACK_LISTED_HREF_PREFIX = ['mailto:', 'tel:', 'sms:'];
```

Comment:
I think we are avoiding the term black list (let me know if you have questions about this)- deny list is one good substitute

Also, let's not add tel or sms to the list unless we actually need to. mailto is a good exception, since it's something we actually want. 

Although I'd argue mailto is a little useless since it oftentimes isn't hooked up to a real mail client for users. But let's support it...

Reply:
Removed this list and just changed to check against `mailto:`. 

context: this was requested by Cecile and we have a few places in the repo that already use mailto. Adding this here for future readers if we do change this 

https://weightsandbiases.slack.com/archives/C05UN2EG770/p1714173838926099?thread_ts=1713999891.755749&cid=C05UN2EG770